app/cdap/components/shared/ConfigurationGroup/utilities/index.ts

```diff
@@ -389,7 +389,8 @@ export function removeFilteredProperties(values, filteredConfigurationGroups) {
     filteredConfigurationGroups.forEach((group) => {
       group.properties.forEach((property) => {
         if (property.show === false) {
-          delete newValues[property.name];
+          // instead of deleting the field, set it to null
```

Comment:
Do we know why it wants to delete it? One of the unit tests specifically wants it to be deleted